test/action_controller/serialization_test.rb

```diff
@@ -171,7 +171,7 @@ def test_render_array_using_custom_root
         with_adapter :json do
           get :render_array_using_custom_root
         end
-        expected = { custom_roots: [{ name: 'Name 1', description: 'Description 1' }] }
+        expected = { custom_root: [{ name: 'Name 1', description: 'Description 1' }] }
```

Comment:
I can see this breaking a lot of apps.. I'm wondering how we can communicate that?  @rails-api/ams